validator/web/handler.go

```diff
@@ -7,10 +7,9 @@ import (
 	"path"
 )
 
-const prefix = "external/prysm_web_ui/prysm-web-ui"
+const prefix = "prysm-web-ui"
 
 // Handler serves web requests from the bundled site data.
-// DEPRECATED: Prysm Web UI and associated endpoints will be fully removed in a future hard fork.
```

Comment:
Probably want to leave this deprecation?

Reply:
i can add it back